### PR TITLE
feat: update registry spec

### PR DESCRIPTION
**Pull request by shiva-rakshith**

### CODE REVIEW

File: API Definitions/openapi_hcx_registry.yml [hunk 1]
```diff
@@ -60,7 +60,45 @@ paths:
...
+        * queryString("queryString")
+
+        * gt(">")
+
+        * lt("<")
+
+        * gte(">=")
+
+        * lte("<=")
+
```
Comment (vrayulu) [inline API Definitions/openapi_hcx_registry.yml:101]: - give some sample usages of the API
- if multiple filters are given, will an "AND" be applied to the filters? specify that also
- don't we have ability to specify "limit" and "offset"

File: API Definitions/openapi_hcx_registry.yml [hunk 2]
```diff
@@ -60,7 +60,45 @@ paths:
     post:
       tags:
         - Registry APIs
-      description: This API is to search for participants in the registry. API returns list of participants matching the input criteria.
+      description: >-
+        This API is to search for participants in the registry. API returns list of participants matching the input criteria.
+
+
+        Search filter supports all the fields apart from the nested fields.
```
Comment (vrayulu) [inline API Definitions/openapi_hcx_registry.yml:67]: what does it mean by nested fields? do provide some more explanation.

File: API Definitions/openapi_hcx_registry.yml [hunk 3]
```diff
@@ -86,7 +124,7 @@ paths:
                         type: array
                         description: List of participants matching with the input search criteria
                         items:
-                          $ref: '#/components/schemas/ParticipantSearchRequest'
+                          $ref: '#/components/schemas/ParticipantUpdateRequest'
```
Comment (vrayulu) [inline API Definitions/openapi_hcx_registry.yml:127]: the search response will have a timestamp and list of participants?

**Comment by vrayulu** [comment]
@shiva-rakshith - please check the comments and send the PR again.

File: API Definitions/openapi_hcx_registry.yml (hunk 1 above)
Comment (shiva-rakshith) [reply API Definitions/openapi_hcx_registry.yml:102]: API supports the limit and offset, I have added the description about them and added examples of API usage.

File: API Definitions/openapi_hcx_registry.yml (hunk 3 above)
Comment (shiva-rakshith) [reply API Definitions/openapi_hcx_registry.yml:172]: yes

File: API Definitions/openapi_hcx_registry.yml (hunk 2 above)
Comment (shiva-rakshith) [reply API Definitions/openapi_hcx_registry.yml]: Initially did not find any reference on how to use nested fields in filters and tried a few ways, but no luck. 
Now found a way to use nested fields in the filters, and I have added it to examples.

**Comment by shiva-rakshith** [comment]
@vrayulu - Fixed all the feedback comments. Please review.

**Pull request merged**